include/TPP/Transforms/Utils/VNNIUtils.h

```diff
@@ -19,7 +19,7 @@ class MemRefType;
 namespace vnni {
 namespace utils {
 
-enum class VnniOp { TRANSPOSE = 3, GEMM = 3, BRGEMM = 4 };
+enum class VnniOp { TRANSPOSE = 3, GEMM = 3, BRGEMM_INS = 4, BRGEMM_OUTS = 3 };
```

Comment:
`BRGEMM_OUTS` should be 5?
If not let's use more generic name for value `3`

Reply:
It is the expected VNNI rank, it is 3 for the BRGEMM output operand. Maybe I can rename the struct to something like or got back to an int64_t.
```
struct ExpectedVnniRank {
 THREE = 3,
  FOUR = 4
};
```
For now the idea was to be explicit on the operation we are dealing with and not the rank.

Reply:
Renamed.